sites/public/pages/applications/financial/income.tsx

```diff
@@ -87,7 +87,7 @@ const ApplicationIncome = () => {
     if (!validationError) {
       const toSave = { income, incomePeriod }
 
-      conductor.completeSection(3)
+      conductor.completeSection(4)
```

Comment:
I think we need to make this dependent on whether or not there are programs like we do on line 65